BaseClasses.py

```diff
@@ -347,6 +347,8 @@ def find_common_pool(players: Set[int], shared_pool: Set[str]) -> Tuple[
             for item in self.itempool:
                 count = common_item_count.get(item.player, {}).get(item.name, 0)
                 if count:
+                    if ItemClassification.progression not in item.classification:
```

Comment:
Does this not work for some reason?
```suggestion
                    if not item.advancement:
```

Reply:
just forgot about it lol